tests/test_refObjLoader.py

```diff
@@ -0,0 +1,294 @@
+        loaderConfig.filterMap = {'aprime': 'a'}
+        loader = ReferenceObjectLoader([dataRef.dataId for dataRef in self.datasetRefs],
+                                       self.handles,
+                                       config=loaderConfig)
+        center = lsst.geom.SpherePoint(180.0*lsst.geom.degrees, 0.0*lsst.geom.degrees)
+        cat = loader.loadSkyCircle(
+            center,
+            30.0*lsst.geom.degrees,
+            filterName='a'
+        ).refCat
+
+        # Zero epoch change --> no proper motion correction (except minor numerical effects)
+        cat_pm = loader.loadSkyCircle(
+            center,
+            30.0*lsst.geom.degrees,
+            filterName='a'
+        ).refCat
+        loader.applyProperMotions(cat_pm, self.epoch)
+
+        self.assertFloatsAlmostEqual(cat_pm['coord_ra'], cat['coord_ra'], rtol=1.0e-14)
+        self.assertFloatsAlmostEqual(cat_pm['coord_dec'], cat['coord_dec'], rtol=1.0e-14)
+        self.assertFloatsEqual(cat_pm['coord_raErr'], cat['coord_raErr'])
+        self.assertFloatsEqual(cat_pm['coord_decErr'], cat['coord_decErr'])
+
+        # One year difference
+        loader.applyProperMotions(cat_pm, self.epoch + 1.0*astropy.units.yr)
+        self.assertFloatsEqual(cat_pm['pm_raErr'], cat['pm_raErr'])
+        self.assertFloatsEqual(cat_pm['pm_decErr'], cat['pm_decErr'])
+        for orig, ref in zip(cat, cat_pm):
+            self.assertAnglesAlmostEqual(orig.getCoord().separation(ref.getCoord()),
+                                         self.properMotionAmt, maxDiff=1.0e-6*lsst.geom.arcseconds)
+            self.assertAnglesAlmostEqual(orig.getCoord().bearingTo(ref.getCoord()),
+                                         self.properMotionDir, maxDiff=1.0e-4*lsst.geom.arcseconds)
+        predictedRaErr = np.hypot(cat["coord_raErr"], cat["pm_raErr"])
+        predictedDecErr = np.hypot(cat["coord_decErr"], cat["pm_decErr"])
+        self.assertFloatsAlmostEqual(cat_pm["coord_raErr"], predictedRaErr)
+        self.assertFloatsAlmostEqual(cat_pm["coord_decErr"], predictedDecErr)
+
+    def test_requireProperMotion(self):
+        """Tests of the requireProperMotion config field."""
+        loaderConfig = ReferenceObjectLoader.ConfigClass()
+        loaderConfig.requireProperMotion = True
+        loader = ReferenceObjectLoader([dataRef.dataId for dataRef in self.datasetRefs],
+                                       self.handles,
+                                       config=loaderConfig)
+        center = lsst.geom.SpherePoint(180.0*lsst.geom.degrees, 0.0*lsst.geom.degrees)
+
+        # Test that we require an epoch set.
+        msg = 'requireProperMotion=True but epoch not provided to loader'
+        with self.assertRaisesRegex(RuntimeError, msg):
+            _ = loader.loadSkyCircle(
+                center,
+                30.0*lsst.geom.degrees,
+                filterName='a'
+            ).refCat
+
+        # Test that anyFilterMapsToThis works.
```

Comment:
Ah yes, I have confirmed that it is.  I've removed these redundant tests (anyFilter and below), but left the config test in since that is otherwise not covered (hitting https://github.com/lsst/meas_algorithms/blob/f36ae6ace101d9874c6f103cd9182f19e188b404/python/lsst/meas/algorithms/loadReferenceObjects.py#L281 )